x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/edit_policy.tsx

```diff
@@ -333,7 +333,7 @@ export const EditPolicy: React.FunctionComponent = () => {
 
         {isShowingPolicyJsonFlyout ? (
           <PolicyJsonFlyout
-            policyName={saveAsNew ? currentPolicyName : policyName}
+            policyName={saveAsNew || isNewPolicy ? currentPolicyName : policyName}
```

Comment:
[nit] It looks like this was pre-existing, so this is probably out of scope for this PR 😄 , but without much context, I think it's a little unclear the difference between `saveAsNew` and `isNewPolicy`. Maybe something like `isClonedPolicy` and `isNewPolicy` might be better. Similar concern for `currentPolicyName` vs. `policyName`, but again, I understand this is unrelated to your changes.